packages/flutter_tools/lib/src/version.dart

```diff
@@ -40,12 +43,24 @@ enum Channel {
 
 // Beware: Keep order in accordance with stability
 const Set<String> kOfficialChannels = <String>{
-  globals.kDefaultFrameworkChannel,
+  'master',
   'main',
   'beta',
   'stable',
 };
 
+const Map<String, String> kChannelDescriptions = <String, String>{
+  'master': 'tip of tree, for contributors',
```

Comment:
I've found several contributors confused by the term "tip of tree", or "tree" in general with regards to a repository. What about: "latest development branch, for contributors"?

Reply:
done